src/som/primitives/ActivitySpawn.java

```diff
@@ -111,9 +111,13 @@ public final SpawnPrim initialize(final VM vm) {
       this.forkJoinPool = vm.getForkJoinPool();
       this.processesPool = vm.getProcessPool();
       this.threadPool = vm.getThreadPool();
+      thrower = new ExceptionSignalingNode("NotAValue",
```

Comment:
We might want factory methods for these common exceptions.
Having all these magic strings all over the codebase is likely not ideal.

Reply:
Done.